src/System.Data.Odbc/src/System/Data/Odbc/OdbcStatementHandle.cs

```diff
@@ -4,6 +4,7 @@
 
 using System.Data.Common;
 using System.Runtime.InteropServices;
+using static Interop;
```

Comment:
We've had discussions about this in the past, e.g. https://github.com/dotnet/corefx/issues/6408, and decided not to do it, so we don't do it elsewhere in corefx.  All of the call sites should be `Interop.Odbc.Whatever`.